src/main/java/com/bookshop/service/impl/BookServiceImpl.java

```diff
@@ -41,14 +41,15 @@ public BookDto getById(Long id) {
 
     @Override
     public BookDto update(Long id, CreateBookRequestDto requestDto) {
-        getById(id);
+        checkIsPresent(id);
         Book newBook = bookMapper.toModel(requestDto);
         newBook.setId(id);
         return bookMapper.toDto(bookRepository.save(newBook));
     }
 
     @Override
     public void deleteById(Long id) {
+        checkIsPresent(id);
```

Comment:
Generally it's OK if book is not found, in this case nothing will happen. (but if your requirements tell to throw an exception then OK)

Reply:
I think, user should know, if deleting went wrong